javascript/javascript2/week1/shortestword.js

```diff
@@ -0,0 +1,13 @@
+const danishWords = ['bil', 'plante', 'kaffe', 'bob', 'ø', 'planetarium'];
+
+function findShortestWord(shortestWord) {
+	let shortestDanishWord = '';
+	for (let word of shortestWord) {
+		if (typeof word === 'string' && (shortestDanishWord === '' || word.length < shortestWord.length)) {
```

Comment:
Your code doesn't work if I pass the array bellow. Can you see why?
`const danishWords = ['planetarium','bil'];`